.github/workflows/unit-test.yml

```diff
@@ -78,6 +78,9 @@ jobs:
     runs-on: ubuntu-latest
     timeout-minutes: 15
     steps:
+      - uses: actions/setup-go@v3
+        with:
+          go-version: 1.18
```

Comment:
Could you please set go version from using the env var like the unit tests job?

Reply:
Fixed